pgxn/neon/walproposer_pg.c

```diff
@@ -232,6 +245,123 @@ nwp_register_gucs(void)
 							PGC_POSTMASTER,
 							0,
 							NULL, NULL, NULL);
+	DefineCustomStringVariable(
+							   "neon.safekeeper_sslcert",
```

Comment:
nit: the alignment is strange. How is the number of leading spaces determined?

Reply:
I'm just following the previous pattern. I don't like it either. I would rather us use something like clang-format.